frontend/src/components/cart-page/CartItemsSection.tsx

```diff
@@ -0,0 +1,50 @@
+import { useContext } from "react";
+import { useNavigate } from "react-router-dom";
+
+import { ShoppingCartContext } from "../../contexts/shopping-cart-context/shoppingCartContext";
+import { Card, CardContent, Typography } from "@mui/material";
+
+import cartStyles from "./cartStyles";
+import ItemsList from "../checkout/place-order/order-itmes/ItemsList";
+import EmptyCartMessage from "./EmptyCartMessage";
+
+const { cartCardStyle } = cartStyles;
+
+const CartItemsSection = () => {
+    const navigate = useNavigate();
+    const { cartItems } = useContext(ShoppingCartContext);
```

Comment:
Maybe a custom hook would be easier to reuse it across the project. 💭 
```
export const useShoppingCartContext = (): ShoppingCartContext => {
  const cartItems = useContext(ShoppingCartContext);

  if (!cartItems) {
    throw new Error('not within shopping cart provider');
  }

  return cartItems;
};
```
this approach encapsulates the logic and makes it reusable across multiple components, so if you require reusability and explicit error handling, the custom hook approach is better to follow.

Reply:
True, i was thinking about that earlier today and wrote it down as a note to do it in a separate branch later.